CHANGELOG.md

```diff
@@ -3,6 +3,119 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.29](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.28...v4.0.0-alpha.29)
+
+_May 18, 2021_
+
+Big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🇸🇰 Add skSK locale (#1634) @martinvysnovsky
```

Comment:
Most developers won't care

```suggestion
```

We could talk about the improvement of performance instead or how we are making progress with the breaking changes for v4.0.0-beta.0 (% of progress).

Reply:
I mentioned the performance improvement and the consistency changes in the API. I would prefer not to add the % we have in https://github.com/mui-org/material-ui-x/milestone/6 because it has only a few issues and any new issue added to it will inflate the progress.

Reply:
> it has only a few issues and any new issue added to it will inflate the progress.

True, the second order benefit I had in mind is this. One of the objectives of Q2 is to reach the stable state. We are mid-Q2. This would incentive us to rigorously list all the breaking changes we want to make, and commit to them. Any new ones would be deferred to the next major, 6 months after.